info/release-notes/24.5.0-readme.txt

```diff
@@ -103,7 +103,7 @@ Bugfix
     https://github.com/mytourbook/mytourbook/discussions/1194
 
 * Tour location profiles
-  The default location profiles were not translated in the last version. Now they are translated but the location names are saved in the configuraiton file C:\Users\<YOUR NAME>\mytourbook\.metadata\.plugins\net.tourbook\tour-location.xml and will not use the translated names. 
+  The default location profiles were not translated in the last version. Now they are translated but the location names are saved in the configuration file C:\Users\<YOUR NAME>\mytourbook\.metadata\.plugins\net.tourbook\tour-location.xml and will not use the translated names. 
```

Comment:
My finger to type the `t` character was faster than the finger which is typing the `i`

This happens also sometimes when typing `if` -> `fi`